main.go

```diff
@@ -1,150 +0,0 @@
-package main
-
-import (
-	"bytes"
-	"context"
-	"encoding/json"
-	"flag"
-	"fmt"
-	"io"
-	"log"
-	"net/http"
-	"net/http/httputil"
-	"os"
-	"strings"
-
-	"github.com/dapr/dapr/pkg/injector/sidecar"
-	sentryConsts "github.com/dapr/dapr/pkg/sentry/consts"
```

Comment:
@mcruzdev let's use these dependencies to get the constants.

Reply:
The dapr project does not contain those constants:

```
daprTrustAnchorsConfigMapKey   string = "dapr-trust-anchors"
daprTrustCertChainConfigMapKey string = "dapr-cert-chain"
daprTrustCertKeyConfigMapKey   string = "dapr-cert-key"
```

This constants is just to create configmap keys name.

We will use only for map the config map keys, what do you think:?

Reply:
perfect.. that makes sense.